website/docs/user-guides/uninstall-litmus.md

```diff
@@ -20,15 +20,18 @@ To disconnect the [ChaosAgent](../getting-started/resources#chaosagents) connect
 To remove the respective components of the ChaosAgents you need to manually delete the created resources of that ChaosAgent.
 
 ```bash
-kubectl delete chaosengine <CHAOSENGINE_IDs> --all -A
-kubectl delete chaosexperiments <CHAOSEXPERIMENTS_IDs> --all -A
-kubectl delete chaosresults <CHAOSRESULTS_IDs> --all -A
+kubectl delete chaosengine <CHAOSENGINE_IDs> --all -<AGENT_NAMESPACE>
```

Comment:
We may also provide like to FAQ in this page or atleast if chaosengines could not be deleted successfully then the finalizer needs to be removed to let the user proceed as he may be stuck in deletion of chaos engines step

Reply:
Have added a NOTE for now, will link it to FAQ soon